src/datasources/asset/types.ts

```diff
@@ -1,55 +1,89 @@
-import { FieldDTO } from '@grafana/data'
 import { DataQuery } from '@grafana/schema'
 
 export interface AssetMetadataQuery extends DataQuery {
+  queryKind: AssetQueryType,
   workspace: string,
   minionIds: string[]
 }
 
-export enum AssetFilterProperties {
-  AssetIdentifier = 'AssetIdentifier',
-  SerialNumber = 'SerialNumber',
-  ModelName = 'ModelName',
-  VendorName = 'VendorName',
-  VendorNumber = 'VendorNumber',
-  AssetName = 'AssetName',
-  FirmwareVersion = 'FirmwareVersion',
-  HardwareVersion = 'HardwareVersion',
-  BusType = 'BusType',
-  IsNIAsset = 'IsNIAsset',
-  Keywords = 'Keywords',
-  Properties = 'Properties',
-  LocationMinionId = 'Location.MinionId',
-  LocationSlotNumber = 'Location.SlotNumber',
-  LocationAssetStateSystemConnection = 'Location.AssetState.SystemConnection',
-  LocationAssetStateAssetPresence = 'Location.AssetState.AssetPresence',
-  SupportsSelfCalibration = 'SupportsSelfCalibration',
-  SelfCalibrationCalibrationDate = 'SelfCalibration.CalibrationDate',
-  SupportsExternalCalibration = 'SupportsExternalCalibration',
-  CustomCalibrationInterval = 'CustomCalibrationInterval',
-  CalibrationStatus = 'CalibrationStatus',
-  ExternalCalibrationCalibrationDate = 'ExternalCalibration.CalibrationDate',
-  ExternalCalibrationNextRecommendedDate = 'ExternalCalibration.NextRecommendedDate',
-  ExternalCalibrationRecommendedInterval = 'ExternalCalibration.RecommendedInterval',
-  ExternalCalibrationComments = 'ExternalCalibration.Comments',
-  ExternalCalibrationIsLimited = 'ExternalCalibration.IsLimited',
-  ExternalCalibrationOperatorDisplayName = 'ExternalCalibration.Operator.DisplayName',
-  IsSystemController = 'IsSystemController'
+export interface AssetUtilizationQuery extends DataQuery {
+  queryKind: AssetQueryType,
```

Comment:
We can explicitly set the type for the interface:

```
queryKind: AssetQueryType.Utilization,
```

Reply:
Let's don't touch, it causes more problems than it fixes